src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import common.Constant;
+import helper.LogHelper;
+import org.slf4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_object.LoginPage;
+
+public class LoginTest extends BaseTest {
+    LoginPage loginPage = new LoginPage();
+    String errorMessage = "There was a problem with your login and/or errors exist in your form.";
```

Comment:
why do we need this variable?

Reply:
this variable to reuse in the next test case